docs/AboutUs.md

```diff
@@ -19,41 +24,41 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Tan Rui Xuan
 
 <img src="images/johndoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Team Lead
-* Responsibilities: UI
+* Role: Developer
+* Responsibilities: Testing, Logic
 
-### Johnny Doe
+### Tan Ying Jie, Dexter
 
 <img src="images/johndoe.png" width="200px">
 
 [[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
 
-* Role: Developer
-* Responsibilities: Data
+* Role: Project Advisor
```

Comment:
Kind of weird I'm a `Project Advisor` in the project I'm working on. Maybe `Lead Developer` would make more sense, but I'll change that in my own AboutUs update later.

Reply:
Ok i won't be touching it in that case